target/linux/realtek/files-5.15/drivers/net/dsa/rtl83xx/common.c

```diff
@@ -1427,7 +1428,7 @@ static int rtl83xx_fib_event(struct notifier_block *this, unsigned long event, v
 			fib_info_hold(fib_work->fen_info.fi);
 
 		} else if (info->family == AF_INET6) {
-			struct fib6_entry_notifier_info *fen6_info = ptr;
+			/*struct fib6_entry_notifier_info *fen6_info = ptr;*/
```

Comment:
we at least should be consistent in our commenting style; probably best to use 'illegal' slashes, as those will trigger other issues later (upstream rejecting it, grep ability for illegal comments etc).